### DISCUSSION TITLE
folders management clarification, how to use module in the app.js file & call data from GraphQL

### DISCUSSION THREAD

**Discussion by tengkuzulfadli** [alpinejs/alpine#2068] [1. Help]
Hi,

I'm still new with Alpine js. I have 4 questions need clarification:

1) The instruction to install using module on the alpine js website is not clear. I added the line of code provided to the app.js file, but it's not working when I test a simple function for toggling a nav menu. Then, i added the alpine js cdn link, it's working. How can I add the import from 'alpine js' code in the app.js file?

2) Related to the above question, I attached a screenshot for my project folders, is the app.js file correctly placed?

3) I tried to test the app.js file by writing a simple Vanilla js code, then referencing to the index.html file using alpine js, it's not working at all. Then, I tried a second time by placing my vanilla js code inside index.html and added the alpine js cdn file, then it's working. But in this project, I want to place all custom js code separately with html files. Was this issue related to the folders management above?

4) How can I call my database inside Graphql to alpine js?

Appreciate your assistance!
![Screenshot (102)](https://user-images.githubusercontent.com/62935899/133373706-2ac438ca-ebd6-4883-b28e-71f0a3d68e39.png)

**Comment by SimoTod** [comment]
Can you post your app.js and index.html? use ``` to delimitate the code so it reads better
